openmetadata-ui-core-components/src/main/resources/ui/src/colors/defaultColors.ts

```diff
@@ -64,8 +64,8 @@ export const defaultColors: ThemeColors = {
     725: '#363636', // custom shade for search icon
     750: '#404040', // custom shade
     800: '#252B37',
-    900: '#101828',
-    950: '#0C111D',
+    900: '#181D27',
+    950: '#0A0D12',
```

Comment:
Source for this change?

Reply:
https://www.figma.com/design/q2HveWEgd55xkItAppFSaO/Design-System?node-id=1023-36350&p=f&m=dev